dist/variables.less

```diff
@@ -27,11 +27,13 @@
 @c2fo-green: #64B145;
 @c2fo-red: #ca3131;
 @c2fo-blue: #147798;
+@c2fo-blue-light: #169CD9;
 @c2fo-orange: #f69844;
+@c2fo-gray-extra-light: #F9F9F9;
 @c2fo-gray-light: #EFEFEF;
 @c2fo-gray-medium: #777777;
 @c2fo-gray-dark: #333333;
-@c2fo-page-background: #F0F4F7;
+@c2fo-page-background: #F2F2F2;
```

Comment:
I noticed our site background colors had diverged, and I hold out hope that if I keep this repository up to date we will consume it outside of our team because i'm a masochist.